### PR TITLE
chore: Add ADR for NWS API

### PR DESCRIPTION
**Pull request by abbyoung**
# SC-1442

## Proposed changes

Adds ADR for using the National Weather Service API for weather widget.

### CODE REVIEW

**Review by gidjin — APPROVED** [review]

**Review by gidjin — COMMENTED** [review]
Suggestion: Please add a link to the `README.md`'s list of ADRs before merging

File: docs/adr/0025-weather-api.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+## Decision Drivers 
+
+* We want to pass in a zip code and retrieve a location-based forecast.
+* We want a low-cost solution that is sustainable.  As the application user base grows, the number of API calls per minute will increase.
+* We want a well-documented API that is developer-friendly and quick to set up.
+
+## Considered Options
+
+* [OpenWeatherMap](https://openweathermap.org/api
+)
```
Comment (jcbcapps) [inline docs/adr/0025-weather-api.md:26]: Nit-pick: looks like the `)` got moved down? Might cause the url to display.

**Review by jcbcapps — APPROVED** [review]
LGTM! Left a (very) minor comment

File: docs/adr/0025-weather-api.md (hunk 1 above)
Comment (abbyoung) [reply docs/adr/0025-weather-api.md:26]: thanks!

**Pull request merged**